### ISSUE TITLE
No way to init with any number?

### ISSUE THREAD

**Issue by sintro** [zbabtkis/NumberFlipper#1]
Looks like I can not initialize the flipper with custom number other than 0. So why do you need this plugin?

**Comment by zbabtkis** [comment]
Sorry, I haven't worked on this for nearly a year, but I plan on refactoring and fixing a few major bugs on this soon. Adding a configuration option for a custom start number shouldn't be hard. I'll make sure that is incorporated into the next release :)

**Comment by sintro** [comment]
Sure, I tried to do it. But imagine how long it will flip to, let say, 9000-th number... No one client can wait for it. I even tried to rewrite some of this plugins code to let it init multi page flipper with number, and it is, but when i do flipTo, everything is broken

**Comment by zbabtkis** [comment]
Yep -- this is buggy and should not be used in any sort of real world application in its current state. It was designed with two digit numbers in mind. I do plan on rethinking the flipping behavior and including bug fixes and features in a large refactor soon.
